ubuntu/files/sources.list

```diff
@@ -37,19 +37,25 @@ deb-src http://archive.ubuntu.com/ubuntu trusty-updates universe
 ## newer versions of some applications which may provide useful features.
 ## Also, please note that software in backports WILL NOT receive any review
 ## or updates from the Ubuntu security team.
-#deb http://archive.ubuntu.com/ubuntu trusty-backports main restricted universe multiverse
-#deb-src http://archive.ubuntu.com/ubuntu trusty-backports main restricted universe multiverse
+#deb http://archive.ubuntu.com/ubuntu {{ grains['oscodename'] }}-backports main restricted universe multiverse
+#deb-src http://archive.ubuntu.com/ubuntu {{ grains['oscodename'] }}-backports main restricted universe multiverse
 
 ## Uncomment the following two lines to add software from Canonical's
 ## 'partner' repository.
 ## This software is not part of Ubuntu, but is offered by Canonical and the
 ## respective vendors as a service to Ubuntu users.
-# deb http://archive.canonical.com/ubuntu trusty partner
-# deb-src http://archive.canonical.com/ubuntu trusty partner
-
-deb http://security.ubuntu.com/ubuntu trusty-security main
-deb-src http://security.ubuntu.com/ubuntu trusty-security main
-deb http://security.ubuntu.com/ubuntu trusty-security universe
-deb-src http://security.ubuntu.com/ubuntu trusty-security universe
-#deb http://security.ubuntu.com/ubuntu trusty-security multiverse
-#deb-src http://security.ubuntu.com/ubuntu trusty-security multiverse
+# deb http://archive.canonical.com/ubuntu {{ grains['oscodename'] }} partner
+# deb-src http://archive.canonical.com/ubuntu {{ grains['oscodename'] }} partner
+
+{% if grains['oscodename'] == 'trusty' %}
```

Comment:
We can make this a little more DRY with Jinja, by putting this at the end of these security lines:
```
{% if grains['oscodename'] == 'xenial' %}restricted{% endif %}
```
This way we only have each line once instead of duplicating them.